src/containers/LocationPage/LocationPage.jsx

```diff
@@ -452,11 +455,16 @@ class LocationPage extends PureComponent {
   }
 
   renderFixedSummaryData() {
+    const { summary = {} } = this.props;
```

Comment:
I think default props is better in general, but when I did this I was wondering if I wanted to handle not having summary data differently (e.g. return null) and so I did this as a temporary fix. I think this presently has the table headers rendered but I'll have to investigate more.